src/js/components/Text/index.d.ts

```diff
@@ -35,6 +37,7 @@ export interface TextProps {
   truncate?: boolean;
   weight?: 'normal' | 'bold' | number;
   wordBreak?: 'normal' | 'break-all' | 'keep-all' | 'break-word';
+  tip?: TipProps;
```

Comment:
Great! Thank you! Can you please also add this new prop to Text/doc.js? 
It will require updating our README and snapshots, so please use `yarn test-update` and push the new files up the branch.